Sources/DockProgress/DockProgress.swift

```diff
@@ -61,6 +61,7 @@ public enum DockProgress {
 	public enum ProgressStyle {
 		case bar
 		case circle(radius: Double, color: NSColor = .controlAccentColorPolyfill)
+		case rectangle(width: Double, height: Double, x: Double, y: Double, color: NSColor = .controlAccentColorPolyfill)
```

Comment:
Let's call it `squircle`, as that's what it's supposed to be.

I think it's enough with just a `radius` parameter here too, and default to the standard macOS 11 radius for icons. 

https://developer.apple.com/design/resources/